x-pack/solutions/security/plugins/security_solution/server/lib/entity_analytics/utils/transforms.ts

```diff
@@ -116,8 +117,15 @@ export const reinstallTransform = async ({
   });
 };
 
-export const getLatestTransformId = (namespace: string): string =>
-  `risk_score_latest_transform_${namespace}`;
+export const getLatestTransformId = (namespace: string): string => {
+  const maxNamespaceLength = 36;
+  const prefix = `risk_score_latest_transform_`;
+  const fullName = `${prefix}${namespace}`;
+
+  const processedNamespace =
+    fullName.length > maxNamespaceLength ? murmurhash.v3(namespace).toString(36) : namespace;
```

Comment:
toString here takes the base as an argument, base 36 does actually work here because that uses 0-9A-Z to represent numbers, but base 16 (hexadecimal) might be a more common choice.